client/src/components/brushableHistogram/index.js

```diff
@@ -650,7 +650,8 @@ class HistogramBrush extends React.PureComponent {
       Number.isNaN(summary.min) ||
       Number.isNaN(summary.max);
 
-    const OK2Render = !summary.categorical && !nonFiniteExtent;
+    const OK2Render =
+      !summary.categorical && !nonFiniteExtent && !isSingleValue;
```

Comment:
Can we make the `isSingleValue` check as early as possible and instead of assigning the value to `OK2Render` just do an early exit with the return value of `{OK2Render: false}` to avoid unused computes